cytosnake/cli/cli_checker.py

```diff
@@ -82,7 +87,7 @@ def cli_check(args_list: list[Union[str, int, bool]]) -> bool:
         for param in args_list
         if param.lower() != "help"
     ]
-    check = len([_bool for _bool in m_bool_mask if _bool == True])
+    check = len([_bool for _bool in m_bool_mask if _bool is True])
```

Comment:
```suggestion
    check = len([_bool for _bool in m_bool_mask if _bool])
```

You need not specify == True for a boolean